### ISSUE TITLE
Is it possible to make it work on an Orange Pi Zero Plus (H5)?

### ISSUE THREAD

**Issue by Benzomat** [vanvught/h3dmx512-zip#6]
Hello @vanvught,

I tried to make the LTC SMPTE Timecode Reader / Converter Generator work on an Orange Pi Zero Plus, with no success. I used the current h3dmx512-zip-master.zip. I flashed the sd card with balenaEtcher, using the image in uboot-orangpi_zero.zip. I copied the file in opi_emac_ltc_smpte.zip to the sd card (root directory, same folder as the file uboot.spi and others), I tried it without renaming it, with removed file extension, and renamed to uImage. I replaced the uboot.spi with the one found in uboot-spi.zip (even though I used the current version of uboot-orangpi_zero.zip).
I tried it several times, but the Orange Pi Zero Plus just didn't show any sign of life.

I tested the Pi and the sd card with a Debian installation, both work.

After that, I tried to build the software with the source files in rpidmx512-master.zip (using the above mentioned Debian installation). I installed packages, which seemed to be missing:

`sudo apt-get install gcc-arm-none-eabi libnewlib-arm-none-eabi make dfu-util uuid-dev`

I tried the following scripts: build_firmware.sh, build_h3-firmware.sh, build_linux.sh.
Unfortunately, none of the scripts installed the LTC SMPTE (crashed with error messages, e. g. arm-none-eabi-ld: cannot find -lspiflashstore).

Is it possible at all to make it work on an Orange Pi Zero Plus (H5)? If yes, how?

I'd be really happy to hear from you.

Kind regards
Benzomat

**Comment by vanvught** [comment]
Hi Benzomat,

The Orange Pi Zero Plus is not compatible with the Orange Pi Zero LTS, hence it won't work.

> I tried the following scripts: build_firmware.sh, build_h3-firmware.sh, build_linux.sh.

The build_firmware.sh is for the now obsolete Raspberry Pi boards. I have only tested this script on an Intel platform.

The build_h3-firmware.sh is for the Orange Pi Zero LTS and for the Orange Pi One. The same for above; I have only tested this script on an Intel platform (Linux/MacOS).

The build_linux.sh should work on any Linux/MacOS platform. It even does work on Cygwin. However, I've tried it on an Allwinner board running Linux(/Armbian).

> Is it possible at all to make it work on an Orange Pi Zero Plus (H5)? If yes, how?

You would need to write a lib-h5 project similar to the lib-h3 project. 

-- Arjan
